ocs_ci/ocs/ui/workload_ui.py

```diff
@@ -0,0 +1,492 @@
+        """
+        Get the list of pvcs.
+
+        Returns:
+            list: The list of pvcs.
+        """
+        return [
+            pvc_capacity_deployment.pvc_obj.name for pvc_capacity_deployment in self
+        ]
+
+    def _delete_pvc_capacity_deployment_from_list(self, deployment_name: str):
+        """
+        Delete the PvcCapacityDeployment object by deployment name.
+        ! deletes deployment only from the list, not from the cluster !
+
+
+        Args:
+            deployment_name (str): The deployment name.
+        """
+        pvc_capacity_deployment = self.get_pvc_capacity_deployment_by_deployment(
+            deployment_name
+        )
+        self.remove(pvc_capacity_deployment)
+        logger.info(f"Deleted entry {str(pvc_capacity_deployment)} from the list")
+
+    def delete_deployment(self, deployment: Deployment):
+        """
+        Deletes the deployment from the cluster and from the list.
+        :param deployment:
+        """
+        logger.info(f"Delete deployment {deployment.name}")
+        for pvc_capacity_deployment in self:
+            if pvc_capacity_deployment.deployment == deployment:
+                pvc_capacity_deployment.deployment.delete(True, False)
+                self._delete_pvc_capacity_deployment_from_list(deployment.name)
+                break
+        else:
+            raise ValueError(
+                f"PvcCapacityDeployment with deployment {deployment.name} not found."
+            )
+
+    def delete_pvc(self, pvc: PVC):
+        """
+        Delete the PVC with or without data
+        and remove the PvcCapacityDeployment object from list,
+        make sure that PV is deleted as well.
+
+        It is designed to delete mounted PVC with mounted PV as we want to test the behavior of management-console
+        :param pvc: PVC object
+        """
+        logger.info(f"Delete pvc {pvc.name}")
+        ocp = OCP()
+        for pvc_capacity_deployment in self:
+            if pvc_capacity_deployment.pvc_obj == pvc:
+
+                pv_obj = pvc.backed_pv_obj
+                # pvc will stack in terminating state because it is mounted
+                pvc.delete(wait=False, force=True)
+                # remove finalizers from the pvc to be able to delete mounted pvc
+                params = '{"metadata": {"finalizers":null}}'
+                try:
+                    ocp.exec_oc_cmd(
+                        f"patch pvc {pvc.name} -p '{params}' -n {pvc.namespace}"
+                    )
+                    logger.info(
+                        "sleep 2 min to allow pvc to be deleted after patching finalizers"
+                    )
+                    time.sleep(60 * 2)
+                except CommandFailed as ex:
+                    if "not found" in str(ex):
+                        logger.info(f"pvc '{pvc.name}' already deleted")
+
+                out = ""
+                try:
+                    out = ocp.exec_oc_cmd(f"get pv {pv_obj.name}", silent=True)
```

Comment:
What do you want to see here? 
you can get all pvs and verify pv X do not exist. Search `get_pv_names()`

Reply:
fixed